services/spar/src/Spar/API/Types.hs

```diff
@@ -120,21 +122,32 @@ sparResponseURI = SAML.getSsoURI (Proxy @APISSO) (Proxy @APIAuthResp)
 --   * does not expose routes prefixed with /i/
 --   * handles authorization (adding a Z-User header if requests are authorized)
 --   * does not show the swagger end-point itself
-type OutsideWorldAPI = StripSwagger (StripInternal (StripAuth API))
+--
+-- In addition, we strip the SCIM documentation because SCIM doesn't have
+-- Swagger schemas yet.
+type OutsideWorldAPI =
+    StripSwagger (StripSCIM (StripInternal (StripAuth API)))
 
 -- | Strip the nginz-set, internal-only Z-User header
 type family StripAuth api where
     StripAuth (Header "Z-User" a :> b) = b
-    StripAuth (a :<|> b) = (StripAuth a) :<|> (StripAuth b)
+    StripAuth (a :<|> b) = StripAuth a :<|> StripAuth b
     StripAuth x = x
 
 -- | Strip internal endpoints (prefixed with /i/)
 type family StripInternal api where
     StripInternal ("i" :> b) = EmptyAPI
-    StripInternal (a :<|> b) = (StripInternal a) :<|> (StripInternal b)
+    StripInternal (a :<|> b) = StripInternal a :<|> StripInternal b
     StripInternal x = x
 
+-- | Strip the endpoint that exposes documentation.
 type family StripSwagger api where
-    StripSwagger ("sso" :> "api-docs" :> Get '[JSON] Swagger :<|> b) = StripSwagger b
+    StripSwagger ("sso" :> "api-docs" :> a) = EmptyAPI
     StripSwagger (a :<|> b) = StripSwagger a :<|> StripSwagger b
     StripSwagger x = x
+
+-- | Strip SCIM.
+type family StripSCIM api where
+    StripSCIM ("scim" :> a) = EmptyAPI
```

Comment:
I don't think it's hard, but I'd rather do it as a separate task. It's a nice principle to not put undocumented endpoints into `develop`, though, so I'm rather torn on this. If you have a strong heuristic/rule/something about that, I can reconsider.

Reply:
since you asked: my heuristic is to not merge incomplete, broken features like this one.  :-)

i guess if you take the route out of the servant api type, this won't even be needed, no?  and then you get a type error if you ever enable it and are missing swagger instances?

Reply:
> Was it hard to provide the default instances?

NB: I just tried that (just in case I'm wrong and it turns out to be easy). It's not. Or rather, we can provide default instances and generate very broken docs, but this is worse than no docs.

To do it right, we need to write a custom ToSchema instance for every ToJSON instance we have, and then to test it we need to write sane Arbitrary instances for hscim types (because that's what `validateEveryToJSON` requires), and finally we also need to ask servant-swagger to treat the `SCIM` content-type tag as the `JSON` content-type tag, because otherwise it doesn't even find anything to test.

And then nobody will look at those docs anyway, likely. That's also why I'm not very keen on doing that work.

I will, however, remove the StripSCIM instance and move it to my local stash, though it's less convenient than having it in the code.